### ISSUE TITLE
Incorrect Information in the Documentation

### ISSUE THREAD

**Issue by Liong1976** [MSzturc/obsidian-advanced-slides#28]
First of all, I want to thank you for your hard work making this awesome plugin and providing detailed documentation.

I just spotted a mistake in the information about Grid Component > Row Flow. I think you mistakenly copied the info from the Column Flow to the Row Flow. 
![image](https://user-images.githubusercontent.com/88898230/147306313-bfc86a7a-d4d6-4b2b-8159-e62e4738105f.png)

Another thing, the last sample for Fragments (see below) doesn't work for me. It only shows a blank slide.
```
<grid drag="60 55" drop="5 10" bg="red" frag="1">
60 x 55
</grid>

<grid drag="25 55" drop="-5 10" style=bg="green" frag="2">
25 x 55
</grid>

<grid drag="90 20" drop="5 -10" bg="gray" frag="3">
90  x 20
</grid>
```

**Comment by MSzturc** [comment]
Thx for the feedback, i've corrected the docs on row flow and i've created a better sample for fragments. It will be online with the next release of Advanced Slides.

Until then here the fixed sample for fragments:


<grid drag="60 55" drop="5 10" bg="red">

    ### Fragments:
    + Right
    + Bottom
    </grid>
    
    <grid drag="25 55" drop="-5 10" bg="green" frag="1">
    Right Grid
    </grid>
    
    <grid drag="90 20" drop="5 -10" bg="gray" frag="2">
    Bottom Grid
    </grid>

**Comment by MSzturc** [comment]
fixed with 1.0.1

**Comment by Liong1976** [comment]
Thanks!
